bokeh/plotting/tests/test_figure.py

```diff
@@ -235,6 +237,19 @@ def test_title_kwarg_no_warning(recwarn):
     assert len(recwarn) == 0
 
 
+def test_figure_title_should_accept_title():
+    title = Title(text='Great Title')
+    plot = plt.figure(title=title)
+    plot.line([1, 2, 3], [1, 2, 3])
```

Comment:
@brendancol for future reference. This line is unecessary and adds additional possible future failure modes for this test that don't relate to the thing you're trying to test.